tests/interfaces/test_qnode_autograd.py

```diff
@@ -106,10 +106,6 @@ def circuit(a):
         assert isinstance(res, np.ndarray)
         assert res.shape == tuple()
 
-        # without the interface, the tape is unable to deduce
-        # trainable parameters
-        assert circuit.qtape.trainable_params == {0, 1}
-

```

Comment:
Anything special here?

Reply:
My new interfaces inadvertently fix this 🙂 Not intentional, just a result of the logic being a bit cleaner.